packages/plugin-core/src/WelcomeUtils.ts

```diff
@@ -53,27 +54,25 @@ export function showWelcome(assetUri: vscode.Uri) {
               wsPath = wsPathPrimary;
             } else if (!fs.pathExistsSync(wsPathBackup)) {
               wsPath = wsPathBackup;
+            } else {
+              wsPath = await VSCodeUtils.gatherFolderPath({
+                default: path.join(resolveTilde("~"), "Dendron"),
+                override: {
+                  title: "Path for new Dendron Code Workspace",
+                },
+              });
             }
 
-            /*eslint-disable */
-            // This is a workaround to resolve circular dependency.
-            // TODO: fix importing around the package so that we have control over module loading sequence.
```

Comment:
I don't see a circular dep being reported for this file with the webpack tool, but in case we get an issue during packaging we can try reverting this.

Reply:
confirm that this is no longer the case